### PR TITLE
Fixed navbar overflowing

### LINKED ISSUE #41: [BUG] Nav bar overflowing
### Description
Navigation bar is overflowing horizontally and making site horizontally scrollable.

### Screenshots
Screenshot_2023-12-25-20-30-41-939_com android chrome

### Additional information

_No response_

### LINKED ISSUE #45: [BUG] Navbar isn't responsive
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

Production

### Description

When we open the website on the mobile device navbar overflows the device's width.

### Steps to Reproduce

1. Go to https://repo-rater.eddiehub.io/ page.
2. Open the website in mobile view or open in a mobile device.
3. Check the navbar.

### Screenshots

Screenshot 2023-12-26 at 5 19 44 PM
Screenshot 2023-12-26 at 5 20 03 PM

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

I'd probably fix the responsiveness of the navbar by inspecting the navbar, and fix the flexbox by changing it's  flex direction on relevant breakpoints

### PR DESCRIPTION
**Pull request by sohanpaliyal**
## Fixes Issue
Closes #41 
Closes #45

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -12,11 +12,29 @@ export default function Navbar() {
...
+  const [isMobileMenuVisible, setIsMobileMenuVisible] = useState(false);  
+
   const logout = async () => {
     await account.deleteSession("current");
     setUser(null);
   };
 
+
+  const handleResize = (e) => {
+    console.log('handle resixe ===>')
```
Comment (yogeshpaliyal) [inline src/components/Navbar.js:25]: Please remove the console log

File: src/app/globals.css [hunk 2]
```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+
+body:has(.mobile-navbar) {
+    overflow: hidden;
+}
```
Comment (yogeshpaliyal) [inline src/app/globals.css:8]: Add a blank line at end

**Review by yogeshpaliyal — APPROVED** [review]

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 

We are using tailwind, I think we can use their responsive classes, we don't need all the react code

Documentation https://tailwindcss.com/docs/responsive-design

**Comment by sohanpaliyal** [comment]
> Thank you for the contribution 👍
> 
> We are using tailwind, I think we can use their responsive classes, we don't need all the react code
> 
> Documentation https://tailwindcss.com/docs/responsive-design

i think we 'll gonna need atleast one useState to handle these classes .except that state i removed all the other unnecessary react code

**Comment by eddiejaoude** [comment]
I will have a look tomorrow, can you share screenshots please of the changes in desktop and mobile view

**Comment by sohanpaliyal** [comment]
> I will have a look tomorrow, can you share screenshots please of the changes in desktop and mobile view

image

image

image

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by eddiejaoude** [comment]
I will merge to a temporary branch and fix the conflicts

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**